distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -1129,15 +1139,15 @@ round_min_long = -99999999999999
 
 ## PRIVATE
    Restrict rounding decimal_places parameter.
-check_decimal_places : Integer -> Function
+check_decimal_places : Integer -> Any -> Any
```

Comment:
I would think the type of ~action is Any, since it's just a lazy value, and doesn't take any arguments. If there's no error, then the value of action is returned. In your PR I suggested we change this to:

`check_decimal_places : Integer -> Any -> Any ! Illegal_Argument`